### PR TITLE
Rename master/slave for main/worker

### PR DESCRIPTION
**Pull request by theresa-m**
OpenJ9 dependent: https://github.com/eclipse/openj9/pull/10174

Related to: https://github.com/eclipse/openj9/issues/10071

This is the big one! I have gone through all the files for copyright updates. This should be the remaining replacements other than branch names & build scripts.

Signed-off-by: Theresa Mammarella Theresa.T.Mammarella@ibm.com

### CODE REVIEW

File: example/glue/EnvironmentDelegate.cpp [hunk 1]
```diff
@@ -132,7 +132,7 @@ MM_EnvironmentDelegate::releaseExclusiveVMAccess()
 
 /**
  * Give up exclusive access in preparation for transferring it to a collaborating thread
- * (i.e. main-to-master or master-to-main). This may involve nothing more than
+ * (i.e. collaborator-to-main or main-to-collaborator). This may involve nothing more than
```
Comment (theresa-m) [inline example/glue/EnvironmentDelegate.cpp:135]: This is the only place that was not a direct replacement. Please advise if other terms will make more sense.

**Comment by theresa-m** [comment]
any takers for review?

**Comment by charliegracie** [comment]
@genie-omr build all

**Comment by pshipton** [comment]
The OpenJ9 project is ready for this to be merged.

**Comment by rwy0717** [comment]
@0xdaryl do you want to take a quick look at the compiler changes (don't worry, there are only a few)

**Pull request merged**